soh/src/overlays/actors/ovl_En_Ge1/z_en_ge1.c

```diff
@@ -98,11 +98,8 @@ void EnGe1_Init(Actor* thisx, PlayState* play) {
     EnGe1* this = (EnGe1*)thisx;
 
     // When spawning the gate operator, also spawn an extra gate operator on the wasteland side
-    if (IS_RANDO &&
-        (Randomizer_GetSettingValue(RSK_SHUFFLE_GERUDO_MEMBERSHIP_CARD) ||
-         Randomizer_GetSettingValue(RSK_SHUFFLE_OVERWORLD_ENTRANCES)) &&
-        (this->actor.params & 0xFF) == GE1_TYPE_GATE_OPERATOR) {
-        // Spawn the extra gaurd with params matching the custom type added (0x0300 + 0x02)
+    if (IS_RANDO && (this->actor.params & 0xFF) == GE1_TYPE_GATE_OPERATOR) {
```

Comment:
randomizer hook_handlers already has a block for ACTOR_EN_GE1, should see if this can move here